src/workflow/obs_testing_images.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+Creates the final images related to marginal and conditional independence tests
+that use only observable variables.
+
+To execute this module by itself, navigate at the command line to the project's
+root directory and type: `python -m src.workflow.obs_testing_images`.
+"""
+# Standard modules
+from pathlib import Path
+
+# Third party modules
+import click
+import pandas as pd
+
+# Local modules
+from src import utils
+from src.graphs.conditional_independence_example import EXAMPLE_GRAPH
+import src.testing.observable_independence as oi
+
+# Declare paths to data
+DATA_PATH =\
+    (utils.PROJECT_ROOT / 'data' / 'raw' /
+     'spring_2016_all_bay_area_long_format_plus_cross_bay_col.csv')
+
+# Note the columns of interest in the dataset
+MODE_ID_COL = 'mode_id'
+TIME_COL = 'total_travel_time'
+COST_COL = 'total_travel_cost'
+DISTANCE_COL = 'total_travel_distance'
+
+def create_conditional_independence_testing_results(
+    output_path: str,
+    num_permutations: int=100,
+    permuted_color: str='#a6bddb'
+    ) -> None:
+    # Load the raw data
```

Comment:
Documentation needed or not?

Reply:
Documentation definitely needed but I added it in the #116 branch. It'll be incorporated once I merge that branch into this one.